app/assets/javascripts/component_data/member.js.coffee

```diff
@@ -1,16 +1,15 @@
 @MemberData = flight.component ->
   @after 'initialize', ->
     return if not gon.user
-    channel = @attr.pusher.subscribe("private-#{gon.user.sn}")
 
-    channel.bind 'account', (data) =>
+    @attr.ranger.bind "#{gon.user.sn}.account", (data) =>
       gon.accounts[data.currency.code] = data
       @trigger 'account::update', gon.accounts
 
-    channel.bind 'order', (data) =>
+    @attr.ranger.bind "#{gon.user.sn}.order", (data) =>
```

Comment:
You don't need to add #{gon.user.sn} now, just "order" and "trade", user id is determined by user's jwt token.